reverse_engineering/queries/selectedTablesSubQuery/DatabaseIndexesQueryForRetrievingTheTablesSelectedByTheUser.js

```diff
@@ -0,0 +1,37 @@
+const { QueryForRetrievingTheTablesSelectedByTheUser } = require('./QueryForRetrievingTheTablesSelectedByTheUser');
+
+class DatabaseIndexesQueryForRetrievingTheTablesSelectedByTheUser extends QueryForRetrievingTheTablesSelectedByTheUser {
+	constructor({ schemaToTablesMap }) {
+		super();
+		this.schemaToTablesMap = schemaToTablesMap;
+	}
+
+	getQuery() {
+		const propertiesToSelect = {
```

Comment:
The projection is... the columns to select. So there is something confusing here.